src/data/pfun.lean

```diff
@@ -612,11 +612,7 @@ rel.core_inter _ s t
 
 lemma mem_core_res (f : α → β) (s : set α) (t : set β) (x : α) :
   x ∈ core (res f s) t ↔ (x ∈ s → f x ∈ t) :=
-begin
-  simp [mem_core, mem_res], split,
```

Comment:
Non-terminal `simp` is to blame here, but I suppose writing it this way lead to the proof being removable which is a good thing